internal/integration/integration_test.go

```diff
@@ -46,6 +48,7 @@ func TestMutation(t *testing.T) {
 	defer server.Close()
 	postClient := newRoundtripClient(t, server.URL)
 	getClient := newRoundtripGetClient(t, server.URL)
+	websocketClient := newRoundtripWebScoketClient(t, "ws"+strings.TrimPrefix(server.URL, "http"))
```

Comment:
```suggestion
	websocketClient := newRoundtripWebScoketClient(t, server.URL)
```
pour garder un schéma identique aux autres RoundTrip et mettre le `"ws"+strings.TrimPrefix(server.URL, "http")` dans la fonction.
Et peut etre ajouter un check si les server.URL est déjà au format "ws://..."

Reply:
done